### DISCUSSION TITLE
State of package development

### DISCUSSION THREAD

**Discussion by NoahJ-SK** [Aniketkhote/refreshed#42] [General]
Hi, 

Considering switching a production project over from GetX.
I was wondering how active this package is, and if there is plans by the current maintainers for future improvement independent of GetX. 

Additionally, does the package, like GetX, decouple dependency injection from the other trimmed libraries in refreshed? 

Thanks

**Comment by Aniketkhote** [comment]
Thank you for your interest in Refreshed!

Yes, the project is active, and we’re continually working on improvements. Our current focus is to ensure compatibility, resolve existing issues, and enhance the overall developer experience.

As for decoupling the dependency injection system from other features, it’s not planned at the moment. The internal structure of the library makes it quite complex to decouple without introducing additional challenges. However, we’re committed to making the package more stable and user-friendly, which remains our top priority.

If you have specific requirements or suggestions, feel free to share them! Your input can help us shape the roadmap for future updates.

Looking forward to your feedback!
